frontend/src/components/activePage/TableWorkspaceLogic/TableWorkspaceLogic.tsx

```diff
@@ -148,8 +160,15 @@ const TableWorkspaceLogic: FC<ITableWorkspaceLogicProps> = ({ ...props }) => {
           if (notify && matchNS)
             notifyStatus(instance.status?.phase, instance, updateType);
 
-          const newItem = { ...prev };
-          setDataInstances(newItem);
+          if (
+            objType === SubObjType.Addition ||
+            objType === SubObjType.Deletion ||
+            objType === SubObjType.PrettyName ||
+            objType === SubObjType.UpdatedInfo
+          ) {
```

Comment:
```suggestion
          if (objType !== SubObjType.Drop) {
```
I saw this check in many other places... I wonder if it can't be simplified in this way everywhere...